src/mito2/src/worker.rs

```diff
@@ -398,12 +401,15 @@ impl<S: LogStore> RegionWorkerLoop<S> {
         for ddl in ddl_requests {
             let res = match ddl.request {
                 DdlRequest::Create(req) => self.handle_create_request(ddl.region_id, req).await,
+                DdlRequest::Drop(_) => todo!(),
                 DdlRequest::Open(req) => self.handle_open_request(ddl.region_id, req).await,
                 DdlRequest::Close(_) => self.handle_close_request(ddl.region_id).await,
-                DdlRequest::Alter(_)
-                | DdlRequest::Drop(_)
-                | DdlRequest::Flush(_)
-                | DdlRequest::Compact(_) => todo!(),
+                DdlRequest::Alter(_) => todo!(),
+                DdlRequest::Flush(_) => {
+                    self.handle_flush_request(ddl.region_id, ddl.sender).await;
+                    continue;
```

Comment:
what's this for?

Reply:
It moves the `ddl.sender`. I can't use the sender after the match expression again so I have to continue here.